oereb_client/static/src/request/extract.js

```diff
@@ -59,13 +59,35 @@ export const sanitizeTopicCode = function (theme) {
   return code;
 };
 
-export const groupRestrictionsByTopic = function (restrictions) {
+export const getParentTheme = function(theme, concernedThemes) {
+  let result = null;
+  concernedThemes.forEach((parent) => {
+    if (parent.Code === theme.Code) {
+      result = parent;
+      return null;
```

Comment:
I'am not completely sure but should this return be here?

Reply:
You are right, it has no effect in forEach loops. I have replaced it with a for loop.